SendTip: fail to reload data after tip was send

Stuck like this after tip was send
![SendTip](https://user-images.githubusercontent.com/9008074/91682415-d76aa880-eb94-11ea-8ba5-020df8a45b79.gif)
```
util.js:24 Uncaught (in promise) Error: Request is cancelled by timeout
vue.runtime.esm.js:1888 Error: Request is cancelled by timeout
```

--- a/src/components/layout/sendTip/SendTip.vue
+++ b/src/components/layout/sendTip/SendTip.vue
@@ -111,12 +111,12 @@ export default {
         .then(async () => {
           await Backend.cacheInvalidateTips().catch(console.error);
           this.clearTipForm();
-          EventBus.$emit('reloadData');
           this.$store.dispatch('modals/open', {
             name: 'success',
             title: this.$t('components.layout.SendTip.SuccessHeader'),
             body: this.$t('components.layout.SendTip.SuccessText'),
           });
+          setTimeout(() => EventBus.$emit('reloadData'), 5000);
         }).catch((e) => {
           this.sendingTip = false;
           if (e.code && e.code === 4) {

--- a/src/utils/util.js
+++ b/src/utils/util.js
@@ -21,7 +21,7 @@ export const wrapTry = async (promise) => {
         return null;
       }),
       new Promise(((resolve, reject) => {
-        setTimeout(reject, 10000, new Error('Request is cancelled by timeout'));
+        setTimeout(reject, 12000, new Error('Request is cancelled by timeout'));
       })),
     ]);
   } catch (err) {

